src/admin-views/settings/featured.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
+ * @var string                       $title             Featured settings title.
+ * @var string                       $description       Featured settings description/HTML.
+ * @var string                       $content_template  Template used for the content section.
+ * @var array                        $content_context   Context for template used for content.
+ * @var string[]                     $classes           Array of classes.
+ * @var array                        $links             Array of arrays for links.
+ */
+
+$default_classes = ['tec-tickets__admin-settings-featured'];
+
+?>
+<div <?php tribe_classes( array_merge( $default_classes, (array) $classes ) ); ?> >
+    <?php $this->template( 'settings/featured/title' ); ?>
+    <?php $this->template( 'settings/featured/description' ); ?>
```

Comment:
On the Main class for Featured Settings I made a comment talking about moving the main folder to `settings/featured` which allows use to avoid having to the repeat this part.